js/main.js

```diff
@@ -0,0 +1,105 @@
+let books = null;
+
+function displayBooks() {
+  const bookCollection = document.getElementById('book-list');
+  const list = document.createElement('ul');
+  books.forEach((book) => {
+    const { title } = book;
+    const { author } = book;
+    const { id } = book;
+    const liId = `li${title}`;
+    const bookCard = `<li id=${liId}>
+        <div class="text">
+        <p>"${title}" by</p>
+        <p>: ${author}</p>
+        </div>
+        <button id="btn${id}" class="remove button">Remove</button>
+        </li>
+        <hr>`;
+    list.insertAdjacentHTML('beforeend', bookCard);
+  });
+  bookCollection.innerHTML = '';
+  bookCollection.appendChild(list);
+}
+
+function updateLocalStorage(remove) {
+  if (!remove) {
+    if (books === null) {
+      books = JSON.parse(window.localStorage.getItem('books'));
+    }
+  }
+  window.localStorage.setItem('books', JSON.stringify(books));
+  if (books === null) {
+    books = [];
+  }
+  displayBooks();
+  // eslint-disable-next-line no-use-before-define
+  eventListeners();
+}
+
+function removebook(id) { // eslint-disable-line no-unused-vars
+  const temp = [];
+  let update = false;
+  books.forEach((book) => {
+    if (book.id != id) { // eslint-disable-line eqeqeq
+      if (!update) {
+        temp.push({
+          id: (book.id - 1),
+          title: book.title,
+          author: book.author,
+        });
+      } else {
+        temp.push(book);
+      }
+    } else {
+      update = true;
+    }
+  });
+  books = temp;
+  updateLocalStorage(true);
+}
+
+function eventListeners() {
+  const removeBtn = document.querySelectorAll('.remove');
+  removeBtn.forEach((btn) => {
+    btn.onclick = (event) => {
+      const { target } = event;
+      const { id } = target;
+      const bookId = (id).replace('btn', '');
+      removebook(bookId);
+    };
+  });
+}
+
+function addBook() {
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+  const error = document.getElementById('error');
+  let status = false;
+  if (title === '') {
+    error.innerHTML = 'title is Required';
+  } else {
+    books.forEach((book) => {
+      if (book.title === title && book.author === author) {
```

Comment:
- [ ] Kindly change all regular functions to es6 arrow functions. As stated in the [Javascript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=If%20you%20use%20ES6%2C%20use%20object%20destructuring%20to%20get%20the%20values%20from%20an%20object.%20This%20way%20you%20can%20avoid%20repeating%20a%20lot%20of%20code.).